pyface/array_image.py

```diff
@@ -0,0 +1,95 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
```

Comment:
why add these modules to the top level `pyface` namespace when the more relevant `pyface.image` subpackage exists?

Reply:
The `pyface.image` subpackage is all about loading images from zipfile libraries of images.  These modules are at the same level or above `pyface.image_resources`, and so should live at the same level.